pkg/chunked/dump/dump.go

```diff
@@ -103,18 +103,8 @@ func getStMode(mode uint32, typ string) (uint32, error) {
 	return mode, nil
 }
 
-func sanitizeName(name string) string {
-	path := filepath.Clean(name)
-	if path == "." {
-		path = "/"
-	} else if path[0] != '/' {
-		path = "/" + path
-	}
-	return path
-}
-
 func dumpNode(out io.Writer, added map[string]*internal.FileMetadata, links map[string]int, verityDigests map[string]string, entry *internal.FileMetadata) error {
-	path := sanitizeName(entry.Name)
+	path := internal.CleanAbsPath(entry.Name)
```

Comment:
Right, composefs wants these to be absolute paths, so this makes sense.

(That said, it's kind of tempting to lift that and support relative paths too just on the principle that relative paths are IMO more "canonical" to use in use cases like tar)